Kickstarter-iOS/Views/Controllers/DiscoveryNavigationHeaderViewController.swift

```diff
@@ -199,7 +199,7 @@ internal final class DiscoveryNavigationHeaderViewController: UIViewController {
   fileprivate func showFavoriteCategoriesAlert(categoryName: String) {
     let alertController = UIAlertController(
       title: categoryName,
-      message: Strings.discovery_favorite_categories_alert_message(),
+      message: Strings.Access_all_your_favorite_categories_in_the_explore_dropdown(),
```

Comment:
This copy doesn't make sense as an alert after favoriting a category because the only way they can access this favorite button is after they've switched to a favorite category from the dropdown. Was this copy supposed to be somewhere else, like future onboarding? 
Note, the original copy is "Yes! You've saved this as one of your favorite categories."